.github/workflows/build.yaml

```diff
@@ -51,3 +40,62 @@ jobs:
           cache: 'npm'
       - run: npm ci
       - run: npm run build
+      - uses: actions/upload-artifact@v2
+        with:
+          name: dist
+          path: dist
+  e2e:
+    runs-on: ubuntu-latest
+    needs:
+      - build
+      - lint
+      - test
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: "0"
+      - uses: actions/download-artifact@v2
+        with:
+          name: dist
+          path: dist
+      - uses: cypress-io/github-action@v2
+        with:
+          start: npm run http-server
+      - uses: actions/upload-artifact@v2
+        with:
+          name: e2e-videos
+          path: cypress/videos
+  docker:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
```

Comment:
Is this really needed?
with the `username: ${{ github.repository_owner }}` this shouldn't be needed

Reply:
I remove it.